spikingjelly/activation_based/triton_kernel/flexsn/template.py

```
@@ -125,30 +181,25 @@ def flexsn_{kernel_type}_kernel_{hash}(
 def get_flexsn_inference_kernel(
     core_str: str, core_name: str, info: FlexSNInfo, verbose: bool = False
 ):
-    hash = core_name[-8:]
+    hash = f"{core_name[-8:]}_{INFERENCE_AUTOTUNE_TAG}"
```

Ruff (0.15.12)

[error] Variable `hash` is shadowing a Python builtin

(A001)